doc/sfsmount.1.adoc

```diff
@@ -204,6 +204,11 @@ Equivalent to *--nostdopts* (*-n*) option for use in fstab.
 *-o nonempty*::
 Equivalent to *--nonempty* option for use in fstab.
 
+*-o sfsignoreflush*::
+DANGEROUS. Ignore flush usual behavior by replying SUCCESS to it 
```

Comment:
It is strange to see DANGEROUS here. Probably we can change it to:
Advanced: use with caution.

Reply:
maybe we can reuse the same text without DANGEROUS... from below:

Ignore flush usual behavior by replying SUCCESS to it immediately. Targets fast creation of small files, but may cause data loss during crashes."

Reply:
Done.